StabilityMatrix.Avalonia/ViewModels/CheckpointsPageViewModel.cs

```diff
@@ -832,6 +832,70 @@ public async Task ImportFilesAsync(IEnumerable<string> files, DirectoryPath dest
             .FirstOrDefault(x => x.Path == destinationFolder.FullPath);
     }
 
+    public async Task MoveBetweenFolders(List<CheckpointFileViewModel>? sourceFiles, DirectoryPath destinationFolder)
+    {
+        if (sourceFiles != null && sourceFiles.Count() > 0)
```

Comment:
Since this is a List you can use something like `if (sourceFiles is { Count: > 0 })` instead of the LINQ count, but not a huge deal.